recipe/meta.yaml

```diff
@@ -52,17 +55,14 @@ requirements:
     - setuptools-git >=1.2
     - python-dateutil >=2.8.0
     - tqdm >=4.36.1
-    - {{ pin_compatible('numpy') }}
 
 test:
   imports:
     - prophet
     - prophet.tests
   commands:
-    - pip check
     - py.test -v --pyargs prophet
   requires:
-    - pip
     - pytest
```

Comment:
I think the plan is for the next upstream version to not require PyStan at all, so this can make a return later. At the moment it would fail both because pystan is missing and because upstream has too tight a bound in cmdstanpy